tools/fileinfo/features/vmprotect-heuristic/test.py

```diff
@@ -24,6 +24,7 @@ def test_correctly_analyzes_input_file(self):
             self.fileinfo.output['Detected tool'],
             [
                 'VMProtect (packer), combined heuristic',
-                'Microsoft Linker (12.0) (linker), combined heuristic'
+                'Microsoft Linker (12.0) (linker), combined heuristic',
+                'Microsoft (linker), dos header style'
```

Comment:
Does this change belong to this pull request or is it 2 in 1 pack?

Reply:
Indeed. The change seems to be from https://github.com/avast/retdec-regression-tests/pull/33: 
![diff](https://user-images.githubusercontent.com/5675953/61229983-77633980-a729-11e9-97ed-406879409675.png)

Reply:
Probably a mistake from not git-pull-ing `master` when creating this branch. But anyway - the file is identical to master, not sure why does GIT even bother to report this?

Reply:
GitHub shows it because the change is in a commit on the branch. In other words, it shows changes in commits, not changes compared to `master`. If you remove the change from the commit (via interactive rebase), GitHub will no longer show it.